src/Nethermind/Nethermind.Runner/configs/denver.cfg

```diff
@@ -0,0 +1,47 @@
+{
+  "Init": {
+    "GenesisHash": "0x203e164cf3b6f6765abf2f0355a4d09e5eaf6a777243bc4407b569431cd95cb3",
+    "ChainSpecPath": "chainspec/denver.json",
+    "MemoryHint": 768000000
+  },
+  "Sync": {
+    "FastSync": true,
+    "FastBlocks": true,
+    "UseGethLimitsInFastBlocks": false,
```

Comment:
I don't really know what any of this options mean, please review to something that makes sense to you